### ISSUE TITLE
Time zone issues

### ISSUE THREAD

**Issue by estiens** [estiens/world_cup_json#61]
We are now pulling in UTC times but still parsing them with the Time Zone of the stadium, need to use raw UTC across the app

**Comment by ericman93** [comment]
is this way `http://worldcup.sfg.io/matches/current` isn't working properly?

**Comment by estiens** [comment]
Nope, it is working properly now. The actual FIFA site hasn't updated yet this morning yet!

**Comment by estiens** [comment]
FIFA updated, and we updated :)

**Comment by ericman93** [comment]
is FIFA always that late?

**Comment by ericman93** [comment]
If you know today's games, why isn't current games route query today's games with a time filter?

**Comment by estiens** [comment]
It queries the current match by the one that is actually in progress according to FIFA -- current will always show the match that is /actually live/ at the moment, `/matches/today` will show the ones that are /scheduled/ for today

**Comment by estiens** [comment]
FIFA is not always that late now, you can see it is updating in almost real time now, for some reason it took until game minute 4 before they updated their own official site

**Comment by ericman93** [comment]
isn't to safe to say that every game that is scheduled will actually start at the time?

**Comment by estiens** [comment]
Not nec -- weather delays, other delays, etc. You can get matches scheduled by time pretty easily, but that endpoint is matches actually 'in_progress'

**Comment by nolaneo** [comment]
@estiens thanks for the API!

Are the match times off by 1 hour?
This is what I'm getting for Spain v Portugal:
`datetime: 2018-06-15T12:00:00.000-05:00`

The match is on at 21:00 UTC+3 (or 18:00 UTC), but this datetime would suggest the match is on at (17:00 UTC)

Related to this issue?

**Comment by estiens** [comment]
Yes, I think the datetime of the server was changed before and after I tried to fix it. It looks like all times are off by one hour, should be an easy fix. Or else a location time zone got parsed wrong, I'll double check.

**Comment by estiens** [comment]
Alright, can you give it a spot check now?

**Comment by nolaneo** [comment]
Looks good!
